woocommerce/woocommerce
# Change marketplace install API request to POST instead of GET

### Changes proposed in this Pull Request:

In https://github.com/woocommerce/woocommerce/pull/42952 we introduced a method to retrieve the install url from a Woo.com API. This is failing and that API has now been switched to using `POST` instead of `GET`. This PR updates the request in WooCommerce to a `POST` request to support this change.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch
2. Visit your [My Subscriptions](http://localhost:8888/wp-admin/admin.php?page=wc-admin&tab=my-subscriptions&path=%2Fextensions) page.
3. Ensure you're connected to Woo.com
4. Try to install an available subscription from here
5. Confirm you're able to install a free extension - this should happen on-site
6. Confirm you're able to install a paid extension - this should redirect you to the auto-installer on Woo.com to complete
7. Test around this issue.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Changed an install-url request to POST instead of GET due to changes on Woo.com

</details>
